home-data-migration/src/main/resources/db/changelog/v-1.0/001-create-users-table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE if not exists public.users (
+                            id int not null  primary key,
+                            first_name varchar (255)  not null,
+                            last_name varchar (255)  not null,
+                            login varchar (255) not null,
+                            password varchar (255)  not null,
+                            phone_number varchar (255) not null,
```

Comment:
Do we really have `_phone_number_` in database? Isn't it should be called` contacts`?